orangecontrib/spectroscopy/widgets/owmultifile.py

```diff
@@ -377,16 +420,16 @@ def show_error(li, msg):
                     xs, vals, additional = reader.read_spectra()
                     if additional is None:
                         additional = Table.from_domain(empty_domain, n_rows=len(vals))
-                    data_list.append((xs, vals, additional))
+                    additional.special_spectral_data = xs, vals
```

Comment:
Small nit about the `special_spectral_data` naming (throughout): this data is not usually spectral data but rather non-spectral metadata

Also, I couldn't tell if the additional table handling is tested in this PR? `AsciiColReader` always returns `additional=None`

Otherwise :+1:

Reply:
Thanks!

`special_spectra_data` are the actual wavenumbers and spectra that I just attach to the meta table. I agree the name is funny. :)

Good point on testing the additional data! They probably work as they are handled as normal Orange data in concatenation with the same algorithm as the concatenate uses (multifile does special stuff only for special spectral data), but should be tested still. I'll add a test.